docker/api/exec-pre-app.sh

```diff
@@ -15,8 +15,8 @@ if [ -e "/opt/cnaas/templates" ]; then
     rm -rf /opt/cnaas/templates
 fi
 
-git clone $GITREPO_SETTINGS /opt/cnaas/settings
-git clone $GITREPO_TEMPLATES /opt/cnaas/templates
+#git clone $GITREPO_SETTINGS /opt/cnaas/settings
```

Comment:
For some reason since yesterday I get errors about invalid reference to refs/heads/master when doing clone here first and then pull from API, seems to work if we let API do clone instead